lms/static/sass/features/_learner-profile.scss

```diff
@@ -273,10 +338,15 @@
         .u-field-message {
             @include float(right);
             width: auto;
+
+            .message-can-edit {
+                position: absolute;
+            }
         }
 
         .u-field.mode-placeholder {
             padding: $baseline;
+            margin: $baseline * 0.75;
             border: 2px dashed $gray-l3;
```

Comment:
Currently none of the margin variables are enabled in the edx-bootstrap repo. The only main settings for pixel sizes are for grids and border radii. I'll leave as is for now and we can convert when we update the edx-bootstrap repo.